Content.Server/Mind/Mind.cs

```diff
@@ -247,6 +249,17 @@ public bool TryRemoveObjective(int index)
             _objectives.Remove(objective);
             return true;
         }
+        
+        /// <summary>
+        /// Adds a briefing (i.e. traitor codewords, nuclear code, ...) to this mind
+        /// </summary>
+        public bool TryAddBriefing(string briefing)
+        {
+            if (briefing == string.Empty || briefing == null) return false;
+            Briefing = briefing;
+            return true;
+        }
```

Comment:
you dont need to nullcheck `briefing` since it isnt nullable

does this function need to exist at all? systems can just set the value of `Briefing` themselves

Reply:
yeah you're right

tbh I don't quite get the nullability stuff or those Try*() functions so I err towards overly cautious